src/main/webapp/app/components/account/ActivateTrialFinish.tsx

```diff
@@ -76,7 +67,7 @@ export default class ActivateTrialFinish extends React.Component<{
             } else {
               this.infoMessage = <Redirect to={PAGE_ROUTE.LOGIN} />;
             }
-          }, REDIRECT_TIMEOUT_MILLISECONDS);
+          }, 3000);
```

Comment:
2 seconds seemed to be too short. I think 3 seconds was enough time to read the text and also not too long